static-site/src/components/splash/index.jsx

```diff
@@ -203,8 +204,8 @@ class Splash extends React.Component {
         <BigSpacer/>
 
         <FlexCenter>
-          <Styles.Button to="docs">
-            Read the documentation
+          <Styles.Button to="https://docs.nextstrain.org/en/latest/index.html" target="_blank" rel="noopener noreferrer">
```

Comment:
I don't think we want to change this to open this in a new tab - but perhaps there is a specific reason for changing this?

Reply:
No, you are right, I only think that would make sense if we were considering it an external link.